src/components/Episodes/Episodes.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import uuidv1 from 'uuid/v1';
+import { increment } from '../../redux/actions';
+import { getAnimeEpisodes } from '../../redux/thunk';
+
+
+const mapDispatchToProps = (dispatch) => ({
+  increment: (number) => dispatch(increment(number)),
+  getAnimeEpisodes: () => dispatch(getAnimeEpisodes())
+});
+
+const mapStateToProps = (state) => ({
+  number: state.number,
+  episodes: state.episodes
+});
+
+class ConnectedEpisodes extends Component {
+  constructor() {
+    super();
+    this.state = {
+      loading: true
+    };
+  }
+
+  getSnapshotBeforeUpdate(prevProps) {
+    if (this.props.episodes !== undefined
+      && this.props.episodes !== prevProps.episodes) {
+      this.setState({
+        loading: false
+      });
+    }
+    return null;
+  }
+
+  componentDidUpdate() {
+  }
+
+  componentDidMount = () => {
+    this.props.getAnimeEpisodes();
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <div className={this.state.loading ? 'lds-dual-ring' : ''} />
```

Comment:
add `classnames` library to project and use it for such purposes. Never add conditions in `className` prop. Make JSX as clean as possible.

Reply:
and also, even if you wanna use conditions - make it shorter. You don't need `true ? true : false` condition if the second block has no effect. Just do `true && true` conditions if you don't have `else`.